include/powsybl/iidm/converter/SimpleAnonymizer.hpp

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_CONVERTER_SIMPLEANONYMIZER_HPP
+#define POWSYBL_IIDM_CONVERTER_SIMPLEANONYMIZER_HPP
+
+#include <map>
+
+#include <powsybl/iidm/converter/Anonymizer.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+namespace converter {
+
+class SimpleAnonymizer : public Anonymizer {
```

Comment:
To be discussed: maybe we should introduce the StringAnonymizer class in the util package

Reply:
Discussed : as `StringAnonymizer ` would be used by `SimpleAnonymizer` for now on, its would not be relevant. So it would be refactored later if needed.